src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcService.scala

```diff
@@ -0,0 +1,55 @@
+package io.iohk.ethereum.faucet.jsonrpc
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.{Address, Transaction}
+import io.iohk.ethereum.faucet.FaucetConfig
+import io.iohk.ethereum.faucet.FaucetStatus.FaucetUnavailable
+import io.iohk.ethereum.faucet.jsonrpc.FaucetDomain.{SendFundsRequest, SendFundsResponse, StatusRequest, StatusResponse}
+import io.iohk.ethereum.jsonrpc.{JsonRpcError, ServiceResponse}
+import io.iohk.ethereum.keystore.KeyStore
+import io.iohk.ethereum.mallet.service.RpcClient
+import io.iohk.ethereum.network.p2p.messages.CommonMessages.SignedTransactions.SignedTransactionEnc
+import io.iohk.ethereum.rlp
+import io.iohk.ethereum.utils.{ByteStringUtils, Logger}
+
+import scala.concurrent.Future
+
+class FaucetRpcService(rpcClient: RpcClient, keyStore: KeyStore, config: FaucetConfig) extends Logger {
+
+  private val wallet = keyStore.unlockAccount(config.walletAddress, config.walletPassword) match {
+    case Right(w) => w
```

Comment:
Maybe we could add more logs here? It will be helpful when we will need to debug

Reply:
Done